README.md

```diff
@@ -29,9 +27,10 @@ The commands below are planned to be implemented by product ship date.
   * `up` start an access point with previous credentials
   * `down` stop an access point
 * `tessel logs` lists the recent activity, console.logs of running program from flash or RAM
+* `tessel setup` copies your RSA public key onto the Tessel so that you can SSH into that tessel
+  * `[--name NAME]` sets the hostname of the Tessel
```

Comment:
I would change the language so it's less Linux-y since we're abstracting away the OS. Something like:

```give the Tessel a unique name```